citation/models.py

```diff
@@ -641,9 +641,9 @@ def is_editable_by(self, user):
     def is_archived(self):
         return bool(self.code_archive_url)
 
-    def contributor_data(self):
+    def contributor_data(self, latest=False):
         value = cache.get(CacheNames.CONTRIBUTION_DATA.value + str(self.id))
```

Comment:
this is a good opportunity to refactor Publication and add a `Publication.contribution_data_cache_key` property that returns `"{0}{1}".format(CacheNames.CONTRIBUTION_DATA.value, self.pk)`